### ISSUE TITLE
Dark Color Theme has light input textarea on hover

### ISSUE THREAD

**Issue by zaydogan** [balzack/databag#118]
while typing, the textarea for the input changes to #dddddd, which makes it almost impossible to read.
![image](https://github.com/user-attachments/assets/fb1328d1-23dc-43fd-bb1c-8c385dc351fa)

**Comment by balzack** [comment]
Hmm, it shows up with a black background on my browser, I'll try and reproduce....thanks!

**Comment by zaydogan** [comment]
either the background color, or the text color (which is white) has to change on dark theme :D

**Comment by balzack** [comment]
I agree; it's impossible to read in your screenshot. You can see in my screenshot it looks ok...I'll try some other browsers today.
<img width="852" alt="Screenshot 2024-10-20 at 8 37 17 AM" src="https://github.com/user-attachments/assets/cca0571f-a89a-41e4-adf2-459ceacb9446">

**Comment by zaydogan** [comment]
i am using
https://github.com/balzack/databag/releases/download/v1.1.1019/webapp.tar.gz
is this the latest ?

**Comment by balzack** [comment]
Sorry, that artifact hadn't been updated. I now published the latest build here:
https://github.com/balzack/databag/releases/download/v1.1.1021/webapp.tar.gz

It's the repackaged artifact from the latest integration build:
https://github.com/balzack/databag/actions/runs/11427746568
which includes the regex fix

**Comment by zaydogan** [comment]
yes, it's fixed in 1.1.1021
please close this issue.

**Comment by balzack** [comment]
thank you for your help! I'm sure most people see the issues and just walk away from the project.
